src/components/TargetContainer.js

```diff
@@ -14,7 +14,7 @@ export default Vue.extend({
     const nodes = this.updatedNodes && this.updatedNodes()
     if (!nodes) return h()
     return nodes.length < 2 && !nodes[0].text
-      ? nodes
+      ? h(nodes[0].tag, nodes[0].data, nodes[0].children)
```

Comment:
Hi! So as I just explained in #20, I can't reproduce this locally, which might be due to the version of Vue I'm using vs. people reporting this issue.

Nonetheless, it would be more correct to simply return the single node like this:

```suggestion
      ? node[0]
```

Can you verify wether this works for you?

Reply:
Hello! Thank you for getting back to me.

In the latest version of Vue, this issue appears to no longer be occurring. Unfortunately, even though I went back to the version that was current at the time I opened the PR (2.6.9), and then to 2.6.8, I can't replicate it there anymore either.

I do remember that just returning the single node (as per your suggested change) _didn't_ work, because it was the first thing I tried and I couldn't understand why it didn't work (or why wrapping its data in a call to `h()` _did_).

I'm sorry I can't be more help!